modular_skyrat/modules/automapper/code/area_spawn_entries.dm

```diff
@@ -0,0 +1,159 @@
+SUBSYSTEM_DEF(area_spawn)
+	name = "Area Spawn"
+	flags = SS_NO_FIRE
+
+/datum/controller/subsystem/area_spawn/Initialize(start_timeofday)
+	for(var/iterating_type in subtypesof(/datum/area_spawn))
+		var/datum/area_spawn/iterating_area_spawn = new iterating_type
+		iterating_area_spawn.try_spawn()
+	return ..()
+
+/**
+ * Area spawn datums
+ *
+ * Use these to spawn atoms in areas instead of placing them on a map. It will select any available open and entering turf.
+ */
+/datum/area_spawn
+	/// The target area for us to spawn the desired atom
+	var/list/target_areas
```

Comment:
It would be interesting to add to that documentation that they're in order of priority for the selection of the spawning location (if that's still the case, that is).